sdk_container/src/third_party/coreos-overlay/profiles/coreos/base/package.accept_keywords

```diff
@@ -120,3 +120,11 @@
 
 # Needed by updated app-containers/containers-common
 =sys-fs/fuse-overlayfs-1.14 ~amd64 ~arm64
+
+# azure-keyvault-pkcs11
+=app-crypt/azure-keyvault-pkcs11-0_p20240923 ~amd64
+=app-crypt/p11-kit-0.25.5 ~amd64
+=dev-cpp/azure-core-1.14.0 ~amd64
+=dev-cpp/azure-security-keyvault-certificates-4.2.1 ~amd64
+=dev-cpp/azure-security-keyvault-keys-4.4.1 ~amd64
+=dev-cpp/azure-identity-1.10.0 ~amd64
```

Comment:
Could you keep those entries sorted with the rest of the file? (Or is it something frowned upon in Gentoo? I suppose that Gentoo has conventions we are not really following here.)

Also, I'd add the arm64 keywords too, even if still don't have the arm64 SDK yet.

Reply:
I decided to stabilise the SDK packages since it's been long enough. I'll move the other entries.

As previously mentioned, that's not what `~amd64 ~arm64` actually does. :wink: In any case, we haven't tested any of this on arm64 yet, and it wouldn't make much sense because this tooling is only used in a CI job that will only be run on Azure. Running that as arm64 would just cost more. :laughing:

Reply:
> I decided to stabilise the SDK packages since it's been long enough. I'll move the other entries.

Cool.

> As previously mentioned, that's not what `~amd64 ~arm64` actually does. 😉

Yeah, I need to wrap my head around how this really works. But for atoms like `=foo-bar/pkg-1.2.3 ~amd64 ~arm64` works exactly the way we expect. ;) For other atom forms, it probably different and less obvious.

> In any case, we haven't tested any of this on arm64 yet, and it wouldn't make much sense because this tooling is only used in a CI job that will only be run on Azure. Running that as arm64 would just cost more. 😆

You'd need to add those keywords anyway if you wanted to build the arm64 SDK regardless of whether you want to use those tools or not - I think that building would fail as there wouldn't be any package version available to emerge. Anyway, it's water under the bridge now, because you stabilized them. ;)